RecCalorimeter/src/components/NoiseCaloCellsFromFileTool.cpp

```diff
@@ -1,7 +1,7 @@
 #include "NoiseCaloCellsFromFileTool.h"
 
 // FCCSW
```

Comment:
FCCSW -> k4Package

Reply:
done in most of the files in RecCalorimeter and RecFCCeeCalorimeter. might have a couple of headers to fix yet because I have local changes that I do not want to push at the moment (working on combined ECAL+HCAL clustering) these might go in a separate PR